.github/workflows/docker-build.yaml

```diff
@@ -45,21 +45,21 @@ jobs:
         # Setup docker build to use your Namespace workspace builder
         name: Set up Namespace Buildx
         uses: namespacelabs/nscloud-setup-buildx-action@v0
-      - name: Build runwasi as static muls for amd64 and arm64
-        run: |
-          git clone https://github.com/containerd/runwasi.git &&
-          cd runwasi &&
-          docker build --platform=linux/arm64,linux/amd64 --build-arg BASE_IMAGE=alpine --build-arg CRATE="containerd-shim-wasmtime,containerd-shim-wasmedge,containerd-shim-wasmer" -t nscr.io/a8fcp47vcfori/build-runwasi --push . &&
-          cd -
+#      - name: Build runwasi as static muls for amd64 and arm64
+#        run: |
+#          git clone https://github.com/containerd/runwasi.git &&
+#          cd runwasi &&
+#          docker build --platform=linux/arm64,linux/amd64 --build-arg BASE_IMAGE=alpine --build-arg CRATE="containerd-shim-wasmtime,containerd-shim-wasmedge,containerd-shim-wasmer" -t nscr.io/a8fcp47vcfori/build-runwasi --push . &&
+#          cd -
       - 
         name: Build and push Docker image
         uses: docker/build-push-action@v4
         with:
           context: .
           push: true
           platforms: linux/amd64,linux/arm64
-          build-args: |
-            CONTAINERD_RUNWASI=nscr.io/a8fcp47vcfori/build-runwasi
+#          build-args: |
+#            CONTAINERD_RUNWASI=nscr.io/a8fcp47vcfori/build-runwasi
```

Comment:
If you commented this out, will line 11 in the Dockerfile be affected?

Reply:
Yes, its a switch. when the `CONTAINERD_RUNWASI` image is not provided it uses the artifacts from the `download-containerd-runwasi` stage.